releasenotes/notes/remove_depreciated_0.15_code-ee5c7d343e310888.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    Removed deprecated references to rho in plot_state_hinton,
+    plot_bloch_multivector, plot_state_city, plot_state_paulivec,
+    and plot_state_qsphere.
+    Removed scheduler.utils as all contained functions were moved to
+    pulse/macros and pulse/utils.
```

Comment:
```suggestion
     Removed the argument `rho` in :func:`~qiskit.visualization.plot_state_hinton`, :func:`~qiskit.visualization.plot_bloch_multivector`, :func:`~qiskit.visualization.plot_state_city`, :func:`~qiskit.visualization.plot_state_paulivec`, and :func:`~qiskit.visualization.plot_state_qsphere`. That argument was replaced by `state`.
    Removed `qiskit.scheduler.utils` as all contained functions were moved to
    `qiskit.pulse.macros` and `qiskit.pulse.utils`.
    All these were deprecated since 0.15 (released on August 06, 2020) and now they are removed.
```